api/src/opentrons/protocol_engine/commands/absorbance_reader/close_lid.py

```diff
@@ -108,6 +125,21 @@ async def execute(
             labware_definition_uri=loaded_lid.definitionUri,
             labware_location=new_location,
         )
+
+        if not self._state_view.config.use_virtual_modules:
+            abs_reader = self._equipment.get_module_hardware_api(mod_substate.module_id)
+
+            if abs_reader is not None:
+                result = await abs_reader.get_current_lid_status()
+                if result is not AbsorbanceReaderLidStatus.ON:
+                    raise CannotPerformModuleAction(
+                        "The Opentrons Plate Reader lid mechanicaly position did not match expected Closed state."
```

Comment:
can this error ever bubble up to be shown to users? if so, needs an edit.
```suggestion
                        "The mechanical position of the Absorbance Plate Reader's lid did not match the expected 'closed' state."
```